clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClientConfig.scala

```diff
@@ -1,19 +1,31 @@
 package org.broadinstitute.clio.client
-import org.broadinstitute.clio.util.config.ClioConfig
 
+import java.nio.file.Path
+
+import org.broadinstitute.clio.util.config.{ClioConfig, ConfigReaders}
 import com.typesafe.config.Config
 import net.ceedubs.ficus.Ficus._
-
 import scala.concurrent.duration.FiniteDuration
 
 /**
   * Configuration file accessors.
   */
-object ClioClientConfig {
+object ClioClientConfig extends ConfigReaders {
   private val config = ClioConfig.load
 
+  private val clientConfig = config.as[Config]("client")
+
   val responseTimeout: FiniteDuration =
-    config.as[FiniteDuration]("client.response-timeout")
+    clientConfig.as[FiniteDuration]("response-timeout")
+
+  val greenTeamEmail: String = clientConfig.as[String]("greenteam.email")
+
+  // This has to be checked for null since Config can't handle nulls
+  val serviceAccountJson: Option[Path] =
+    if (clientConfig.getIsNull("service-account-json"))
```

Comment:
I'm still not convinced this is needed... will play around with it locally.

Reply:
Locally, if I replace the whole if-else with `clientConfig.getAs[Path]("service-account-json")` everything works.

Reply:
I did that and it failed on the integration tests.